frontend/packages/knative-plugin/src/utils/create-knative-utils.ts

```diff
@@ -61,6 +62,12 @@ export const createKnativeService = (
     spec: {
       template: {
         metadata: {
+          labels: {
+            'app.kubernetes.io/part-of': applicationName,
```

Comment:
Use `getAppLabels` util function similar to how it's done for other resource creation.

Reply:
We might also need to add user labels that user add from advanced options for label.

Reply:
fixed in https://github.com/openshift/console/pull/2137/commits/f88e7a70a6d69715df9b91e3ef0ccf7c990c6429